src/data/main.py

```diff
@@ -18,51 +20,70 @@
                  "Jadot",
                  "Roussel",
                  "Arthaud",
-                 "Poutou"]
+                 "Poutou",
+                 "all"]
+
 
 # ########################################################################### #
 #                                 Functions                                   #
 # ########################################################################### #
 
 
 def add_data_args(parser):
-    # Specify to the data mentioning a specific candidats or aclImdb
-    parser.add_argument('--download',
-                        default='aclImdb',
+    # Specify which task to perform: download, make_dataset or load
+    parser.add_argument('--task',
+                        required=True,
+                        choices=['download', 'make-dataset', 'load-dataset'],
+                        help="Specify which task to perform.")
+
+    # Specify on which data to perform the task
+    parser.add_argument('--data',
+                        required=True,
+                        default=None,
                         choices=['twitter', 'aclImdb', "allocine"],
-                        help="Download the specified dataset")
+                        help="Specify which type of data on which the task\
+                             will be performed")
 
     # Specify to the data collector the split of the dataset
     parser.add_argument('--split',
-                        default='train',
-                        choices=['train', 'test', "validation"],
-                        help="Download the specified dataset")
+                        default=None,
+                        choices=['train', 'test', "validation", 'predict'],
+                        help="Required For make_dataset: specify the action the \
+                             dataset will be used for ")
 
-    # Specify to the data collector to download tweets mentionning this user
-    parser.add_argument('--text',
+    # Download twitter: specify user
+    parser.add_argument('--user',
                         default=None,
-                        help="Download the tweets from this user")
+                        help="Only for download tweets: Download the tweets\
```

Comment:
The function does not do that.
It downloads tweets mentionning this text

Reply:
ok